provider/pkg/gen/examples/overlays/configGroupV2.md

```diff
@@ -319,195 +319,4 @@ metadata:
 }
 ```
 {{% /example %}}
-{{% example %}}
-### YAML with Transformations
```

Comment:
Can this be moved into the PR related to ConfigGroup V2? Let's try to keep this scoped to just ConfigFile changes.